src/main/java/com/mraof/minestuck/client/gui/AnthvilScreen.java

```diff
@@ -0,0 +1,124 @@
+package com.mraof.minestuck.client.gui;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mraof.minestuck.Minestuck;
+import com.mraof.minestuck.alchemy.*;
+import com.mraof.minestuck.alchemy.recipe.GristCostRecipe;
+import com.mraof.minestuck.blockentity.machine.AnthvilBlockEntity;
+import com.mraof.minestuck.client.util.GuiUtil;
+import com.mraof.minestuck.inventory.AnthvilMenu;
+import com.mraof.minestuck.network.AnthvilPacket;
+import com.mraof.minestuck.network.MSPacketHandler;
+import com.mraof.minestuck.network.MachinePacket;
+import net.minecraft.client.renderer.GameRenderer;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Level;
+import net.minecraftforge.client.gui.widget.ExtendedButton;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.util.List;
+
+@ParametersAreNonnullByDefault
+public class AnthvilScreen extends MachineScreen<AnthvilMenu>
```

Comment:
You don't need to use `MachineScreen` unless you're gonna make use of the `goButton` field (and even then, I'd suggest against it)
```suggestion
public class AnthvilScreen extends AbstractContainerScreen<AnthvilMenu>
```